src/hdfio.cxx

```diff
@@ -1404,9 +1422,17 @@ void ReadHDF(Options &opt, vector<Particle> &Part, const Int_t nbodies,Particle
                     else nchunk=chunksize;
                     for(n=0;n<hdf_header_info[i].npart[k];n+=nchunk)
                     {
-                      if (hdf_header_info[i].npart[k]-n<chunksize&&hdf_header_info[i].npart[k]-n>0)nchunk=hdf_header_info[i].npart[k]-n;
-                      HDF5ReadHyperSlabReal(doublebuff,partsdataset[i*NHDFTYPE+k], partsdataspace[i*NHDFTYPE+k], 1, 1, nchunk, n);
-                      for (int nn=0;nn<nchunk;nn++) Pbaryons[bcount++].SetU(doublebuff[nn]);
+			if (hdf_header_info[i].npart[k]-n<chunksize&&hdf_header_info[i].npart[k]-n>0)nchunk=hdf_header_info[i].npart[k]-n;
+
+			if (opt.ihdfnameconvention != HDFSWIFTFLAMINGONAMES) {
+
+			  HDF5ReadHyperSlabReal(doublebuff,partsdataset[i*NHDFTYPE+k], partsdataspace[i*NHDFTYPE+k], 1, 1, nchunk, n);
+			  for (int nn=0;nn<nchunk;nn++) Pbaryons[bcount++].SetU(doublebuff[nn]);
+
+			} else {
+			  for (int nn=0;nn<nchunk;nn++) Pbaryons[bcount++].SetU(0.);
+			}
+		      }
```

Comment:
I think this is the extra `}` that is causing build problems

Reply:
Yes, indeed, thanks. Wasn't compiling with MPI=OFF when I cleaned up my changes...

I didn't want to run any of the usual code alignment tools as it would change almost every line but it may be interesting to do so one day. (Though it may only be me that is confused by the current alignment strategy and everything is fine for others; it is quite subjective.)

Reply:
I'm absolutely on board with that, I also get confused quite quickly! The only problem is that git history review becomes a tad more difficult (always have to go usually one more step when blaming a line, etc), but the benefits are great. And we would have to enforce the formatting rules, otherwise we'll slowly run into the same issue again.

Reply:
It does definitely create a singularity in the git history. So that can be a problem. Also not easy to port changes from pascal's repo if any are made there and a different formatting is used.

One way to enforce it is to run it also as part of travis and have it complain if the code is unformatted. clang-format can do that and I bet other tools can as well.